modules/system/console/MixRun.php

```diff
@@ -0,0 +1,104 @@
+<?php namespace System\Console;
+
+use File;
+use Symfony\Component\Process\Process;
+use System\Classes\MixAssets;
+use Winter\Storm\Console\Command;
+
+class MixRun extends Command
+{
+    /**
+     * @var string|null The default command name for lazy loading.
+     */
+    protected static $defaultName = 'mix:run';
+
+    /**
+     * @var string The name and signature of this command.
+     */
+    protected $signature = 'mix:run
+        {package : Defines the package to watch for changes}
+        {script : The name of the script to run, as defined in the package.json "scripts" config.}
+        {additionalArgs?* : Arguments to pass through to the }
+        {--f|production : Runs compilation in "production" mode}
+        {--s|silent : Silent mode}';
+
+    /**
+     * @var string The console command description.
+     */
+    protected $description = 'Runs a script in the given package';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): int
+    {
+        $mixedAssets = MixAssets::instance();
+        $mixedAssets->fireCallbacks();
+
+        $packages = $mixedAssets->getPackages();
+        $name = $this->argument('package');
+        $script = $this->argument('script');
+
+        if (!in_array($name, array_keys($packages))) {
+            $this->error(
+                sprintf('Package "%s" is not a registered package.', $name)
+            );
+            return 1;
+        }
+
+        $package = $packages[$name];
+        $packageJson = $this->readPackageJson($package);
+
+        if (!isset($packageJson['scripts'][$script])) {
+            $this->error(
+                sprintf('Script "%s" is not defined in package "%s".', $script, $name)
+            );
+            return 1;
+        }
+
+        $this->info(sprintf('Running script "%s" in package "%s"', $script, $name));
+
+        $command = ($this->argument('additionalArgs')) ?? [];
+        if (count($command)) {
+            array_unshift($command, 'npm', 'run', $script, '--');
+        } else {
+            array_unshift($command, 'npm', 'run', $script);
+        }
+
+        echo `pwd`;
+        echo base_path($package['path']);
```

Comment:
Debugging statements?

Reply:
Yeah, seems to fail in Actions currently, but works locally. :(